tutorials/get-started-kubernetes-storage/index.mdx

```diff
@@ -0,0 +1,247 @@
+
+### Stateless applications vs stateful applications
+
+What kind of applications need to use persistent volumes? To answer this, we must understand the difference between stateless and stateful applications.
+
+**Stateless** applications do not need to load or save data: each request to the application is independent. There may be data in the payload of the request, but has not been retrieved from a previously saved state. A request to a search engine is a good example of this.
+
+**Stateful** applications **do** need to save and load data. Most modern applications are stateless, such as online banking which needs to be able to store and retrieve all your transaction history, and online shopping which stores and remembers your previous orders. In order to run stateful applications in Kubernetes, we need to provision persistent storage volumes where they can store the data that gives them their state.
+
+| Stateless applications                                      | Stateful applications                                            |
+| ----------------------------------------------------------- | -------------------------------------------------------------------- |
+| Do not load or save data                                    | Save and load data                                                   |
+| Each request is independent and doesn't use previous data   | Need to retrieve stored data                                         |
+| Examples: microservice, [whoami](/tutorials/get-started-containers-docker/#example-2-a-more-complex-app), search engine request     | Examples: most modern applications (online banking, online shopping) |
+| "Original" Kubernetes                                       | Kubernetes has now adapted to incorporate statefulness               |
+| Do not need to provision storage                            | Must provision storage                                               |
+
+## How do we provision storage?
+
+### Provisioning Block Storage volumes
+
+Cloud providers such as Scaleway provide and default **StorageClasses** for their managed Kubernetes products. StorageClasses define the classes of storage the provider can offer for the cluster. The user can list the StorageClasses available to by using the command `kubectl get storageclass`.
+
+Scaleway provides [Block Storage](/storage/block/quickstart/) as its StorageClass, meaning that we offer persistent storage for Kapsule clusters via our Block Storage product. 
+
+<Message type="note">
+Scaleway has upgraded the Block Storage used by our StorageClass, offering better performance. Our previous class was `scw-bss`, which has now been replaced by `sbs`, giving you faster read/write operations with lower latency.
```

Comment:
typo, it's `scw-bssd`
now replaced by `sbs-default` to my default sbs only doesn't exist